client/post-editor/media-modal/detail/detail-title.jsx

```diff
@@ -25,6 +25,12 @@ export default React.createClass( {
 		return {};
 	},
 
+	componentWillReceiveProps( nextProps ) {
+		if ( nextProps.item.ID !== this.props.item.ID ) {
```

Comment:
Makes me a bit nervous that `item` is not a required prop yet we compare `ID` property references on it without checking for its existence. I'm guessing it's likely `item` is always an object, though in that case we'd probably want to set the prop as required.